File: 07_Full pyramid/Full_pyramid.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    int i,j,k,rows,itr;

    printf("Enter Rows: ");
    scanf("%d", &rows);

    itr = rows-1;
    
    for(i=0; i<rows; i++){
        for(j=0; j<itr; j++){
            printf("  ");
        }
        for(k=0; k<i+1; k++){
            printf(" *  ");
        }
        itr = itr-1;
        printf("\n");
    }
    return 0;
}

/*
OUTPUT:

               *  
             *   *
           *   *   *
         *   *   *   *
       *   *   *   *   *
     *   *   *   *   *   *
   *   *   *   *   *   *   *
 *   *   *   *   *   *   *   *

*/

File: 07_Full pyramid/full_pyramid.py
rows = int(input("Enter Rows: "))
itr = rows-1

for i in range(0, rows):
    # For printing " "
    for j in range(0, itr):
        print(" ", end=" ")

    # For printing "*"
    for k in range(0, i+1):
        print(" * ", end=" ")
    
    itr = itr-1
    print("")

'''
OUTPUT:
               *
             *   *
           *   *   *
         *   *   *   *
       *   *   *   *   *
     *   *   *   *   *   *
   *   *   *   *   *   *   *
 *   *   *   *   *   *   *   *

'''